desktop/yast2-packager.desktop

```diff
@@ -8,6 +8,7 @@ Terminal=false
 Type=Application
 Categories=System;PackageManager;X-SuSE-ControlCenter-System;
 MimeType=application/x-rpm;application/x-redhat-package-manager;
-NotShowIn=KDE;GNOME;MATE;
+InitialPreference=10
+NotShowIn=GNOME;MATE;
```

Comment:
Sorry, this line cannot go to SP1 as SLES supports only Gnome. This change would cause a regression there..